test/jenkins_tests/run_multi_node_tests.sh

```diff
@@ -11,283 +11,284 @@ ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 DOCKER_SHA=$($ROOT_DIR/../../build-docker.sh --output-sha --no-cache)
 echo "Using Docker image" $DOCKER_SHA
 
-docker run  -e "RAY_USE_XRAY=1" --rm --shm-size=10G --memory=10G $DOCKER_SHA \
+docker run  -e "RAY_USE_XRAY=1" --rm --cpus=12 --shm-size=10G --memory=10G $DOCKER_SHA \
     python /ray/python/ray/rllib/train.py \
     --env PongDeterministic-v0 \
     --run A3C \
     --stop '{"training_iteration": 2}' \
     --config '{"num_workers": 16}'
```

Comment:
Note that setting num CPUs won't don't anything, you have to set num workers. Here it seems to be 16, but could be changed to 2.

Reply:
I think what it should do is limit the total CPU utilization of the docker container (Ray will still create more workers). Does that sound right?

Reply:
I haven't actually verified that.